eth/client.go

```diff
@@ -108,6 +112,42 @@ func (cl *Client) SubscribeToBlocks() (*BlockSubscription, error) {
 	return &BlockSubscription{sub, blocks}, nil
 }
 
+// SubscribeToDeposited writes past Deposited events and newly received ones
+// into the sink. Can be cancelled via context.
+func (cl *Client) SubscribeToDeposited(ctx context.Context, contract *bindings.Erdstall, epochs []uint64, sink chan *bindings.ErdstallDeposited) error {
```

Comment:
So `epochs` is optional to only get specific epochs?

Reply:
As i understand it, it is not optional. go-ethereum will only return deposited events with the specified epochs. I amended the comment.

Reply:
I looked it up, it is optional